src/explore-education-statistics-admin/src/pages/release/footnotes/components/FootnoteForm.tsx

```diff
@@ -124,7 +129,11 @@ const FootnoteForm = ({
             to.
           </p>
 
-          <FormFieldTextArea<BaseFootnote> name="content" label="Footnote" />
+          <FormFieldEditor<BaseFootnote>
+            name="content"
+            label="Footnote"
+            toolbarConfig={['link']}
```

Comment:
I noticed this in #4285, but users will now be able to copy and paste in all sorts of elements from office docs:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/775761bb-b1c6-4688-917c-2a6e93d428ab)

Ideally, we should try and prevent users from doing this, although I can appreciate we're already letting this happen in some other places as well.

To do this, we have two options:

1. Manually disable plugins with the `removePlugins` config option (see [docs](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/configuration.html#removing-features))
2. Create custom CKEditor classes which don't include any of these features in the first place.

**Option 1** is fairly straightforward, but the code isn't too great because we have to do stuff like this:

```ts
import Editor from 'explore-education-statistics-ckeditor';

// later in config...

removePlugins: Editor.builtinPlugins.filter(plugin => plugin.pluginName !== 'Link'),
```

Of course, we'd need to make it more flexible for different use-cases. I think it also breaks existing functionality where we expect the plugin to be installed e.g. comments. Nothing that can't necessarily be fixed though.

**Option 2** would be nicer, but requires a major re-think of how we create the CKEditor editor instances. 

I've also given everything a look and it's currently not looking too good, particularly as we managed to mess up the dependency upgrades and break the CKEditor build by trying to upgrade everything to bleeding edge. 

I gave downgrading a quick test, but there seems to be some other issues with that (probably because we've moved onto Node v18).

I've created [EES-4526](https://dfedigital.atlassian.net/browse/EES-4526) to handle upgrading CKEditor and fixing the build.

Reply:
I've gone with option 1 - you can configure which plugins to include. I've applied this to all the cases where we use the simple toolbar as well, so this will also fix [EES-4513](https://dfedigital.atlassian.net/browse/EES-4513) (I'll close the PR).

It seems that the 'Paragraph' plugin is needed for the editor to work so I'm still sanitising the footnote text as we don't want paragraphs in there.